docs/activities/Development_Guides.mdx

```diff
@@ -768,7 +768,7 @@ let avatarSrc = '';
 if (user.avatar) {
   avatarSrc = `https://cdn.discordapp.com/avatars/${user.id}/${user.avatar}.png?size=256`;
 } else {
-  const defaultAvatarIndex = Math.abs(Number(user.id) >> 22) % 6;
+  const defaultAvatarIndex = (BigInt(user.id) >> 22n) % 6n;
```

Comment:
Ooo lol yeah that makes way more sense. Great catch. We'll want to update this in the sdk example as well.

Reply:
made a PR https://github.com/discord/embedded-app-sdk/pull/150